docs/pages/includes/tctl.mdx

```diff
@@ -17,9 +18,12 @@ $ tctl status
 You can run subsequent `tctl` commands in this guide on your local machine.
 For full privileges, you can also run `tctl` commands on your Teleport Auth Service host.
 
-</ScopedBlock>
+</TabItem>
+</Tabs>
```

Comment:
I don't think this end `</Tabs>` intended. Right?

Reply:
<img width="895" alt="Screenshot 2023-08-18 at 2 04 50 PM" src="https://github.com/gravitational/teleport/assets/12417665/cb666f4a-0c39-4c63-bd13-09511399aae7">

Reply:
@greedy52 Thanks for catching this! I used a script to replace `ScopedBlock`s with `Tabs` components. I thought I caught all of the edge cases, but there are a couple remaining that I'll fix.